ledger/error/src/test/utils/scala/com/daml/error/utils/testpackage/subpackage/MildErrors.scala

```diff
@@ -1,41 +0,0 @@
-// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package com.daml.error.utils.testpackage.subpackage
-
-import com.daml.error._
-import com.daml.logging.LoggingContext
-
-@Explanation("Groups mild errors together")
-object MildErrors
```

Comment:
Moved to MildErrorsParent.scala, where `MildErrors` is a nested within `MildErrorsParent`